packages/cw721/src/traits.rs

```diff
@@ -7,15 +6,10 @@ use crate::{
     AllNftInfoResponse, ApprovalsResponse, ContractInfoResponse, NftInfoResponse,
     NumTokensResponse, OperatorsResponse, OwnerOfResponse, TokensResponse,
 };
-use cosmwasm_std::{Binary, Deps, DepsMut, Empty, Env, MessageInfo, Response, StdResult};
+use cosmwasm_std::{Binary, CustomMsg, Deps, DepsMut, Env, MessageInfo, Response, StdResult};
 use cw_utils::Expiration;
 
-// TODO: move this somewhere else... ideally cosmwasm-std
-pub trait CustomMsg: Clone + std::fmt::Debug + PartialEq + JsonSchema {}
-
-impl CustomMsg for Empty {}
-
-pub trait Cw721<T, C>: Cw721Execute<T, C> + Cw721Query<T>
+pub trait Cw721<T, C>: Cw721Execute<T, C> + Cw721Query<T, C>
```

Comment:
We never actually use the `T` parameter on `Cw721Execute`. Can safely remove it here and in `cw721_base/execute.rs:124` and the build will work fine.

Reply:
I think we need it for mint? Will try removing it and write a test.